packages/store/actions/intl.tsx

```diff
@@ -6,9 +6,9 @@
  * @since:   2017-03-31
  */
 
-import { createSimpleAction } from 'redux-action-helper'
+import { createSimpleAction } from "redux-action-helper"
```

Comment:
If this is just linter fixes, the semicolon should also be included on the imports.

Reply:
resolved